RevenueCatUI/PaywallView.swift

```diff
@@ -306,9 +307,8 @@ struct LoadedOfferingPaywallView: View {
         )
     }
 
-    private func updateSessionIfNeeded() {
+    private func updateSession() {
```

Comment:
Love it

Reply:
Oh `self.session` isn't used anymore outside of `eventData`, which itself is also only used in `onAppear`. Want to remove those 2 properties and just change `eventData` into maybe `createEventData()`?

Reply:
Cause we also don't really need to create the session on `init`.